test/e2e/helpers.js

```diff
@@ -307,6 +321,12 @@ async function withFixtures(options, testSuite) {
         await ganacheServer.quit();
       }
 
+      if (webSocketServer) {
+        webSocketServer.close(() => {
+          console.log('WebSocket server closed');
+        });
```

Comment:
this needs to be awaited

Reply:
I added a promise to the websocket close for it to await

Reply:
This is no longer needed with the mock of websocket requests